IAteIt/IAteIt/Network/FirebaseConnector+ExtensionForComments.swift

```diff
@@ -48,4 +48,9 @@ extension FirebaseConnector {
             }
         }
     }
+    
+    //특정 comment 삭제
+    func deleteComment(commentId: String) async throws {
+        try await FirebaseConnector.comments.document(commentId).delete()
+    }
```

Comment:
```suggestion
    func deleteComment(commentId: String) {
        Task {
            try await FirebaseConnector.comments.document(commentId).delete()
        }
    }
```

async throw 가 들어갈 필요는 없어 보입니다! + commentId를 넘기는 것보다 Comment 자체를 넘기는 편이 좋아보여요